packages/solana_mobile_client/android/build.gradle

```diff
@@ -26,6 +26,7 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdkVersion 31
+    namespace "com.solana.solana_mobile_client_example"
```

Comment:
`com.solana.solana_mobile_client`, not `com.solana.solana_mobile_client_example` ;)